Symfony/CS/Tests/Fixer/Symfony/OperatorsSpacesFixerTest.php

```diff
@@ -159,9 +226,107 @@ public function provideCases()
                 '<?php $a=$b;',
             ),
             array(
+                '<?php $a * $b;',
+                '<?php $a*$b;',
+            ),
+            array(
+                '<?php $a / $b;',
+                '<?php $a/$b;',
+            ),
+            array(
+                '<?php $a % $b;',
+                '<?php $a%$b;',
+            ),
+            array(
+                '<?php $a > $b;',
+                '<?php $a>$b;',
+            ),
+            array(
+                '<?php $a < $b;',
+                '<?php $a<$b;',
+            ),
+            array(
+                '<?php $a | $b;',
+                '<?php $a|$b;',
+            ),
+            array(
+                '<?php $a ^ $b;',
+                '<?php $a^$b;',
+            ),
+            array(
                 '<?php $a = array("b" => "c", );',
                 '<?php $a = array("b"=>"c", );',
             ),
+
+            array(
+                '<?php $a * -$b;',
+                '<?php $a*-$b;',
+            ),
+            array(
+                '<?php $a = -2 / +5;',
+                '<?php $a=-2/+5;',
+            ),
+        );
+    }
+
+    /**
+     * @dataProvider provideCases54
+     * @requires PHP 5.4
+     */
+    public function testFix54($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideCases54()
+    {
+        return array(
+            array(
+                '<?php [1, 2] + [3, 4];',
+                '<?php [1, 2]+[3, 4];',
+            ),
+        );
+    }
+
+    /**
+     * @dataProvider provideCases54
+     * @requires PHP 5.6
+     */
+    public function testFix56($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideCases56()
+    {
+        return array(
+            array(
+                '<?php $a ** $b;',
+                '<?php $a**$b;',
+            ),
+            array(
+                '<?php $a **= $b;',
+                '<?php $a**=$b;',
+            ),
+        );
+    }
+
+    /**
+     * @dataProvider provideCases70
+     * @requires PHP 7.0
+     */
+    public function testFix70($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideCases70()
+    {
+        return array(
+            array(
+                '<?php $a <=> $b;',
+                '<?php $a<=>$b;',
+            ),
```

Comment:
https://github.com/php/php-src/commit/0a4a11b73ae32b31810451d1f7e8719ca0a503db
this commit is created 3 days ago, but travis is using a 7.0 version built 4 days ago. I guess that's the reason. ;)